src/services/sanity.js

```diff
@@ -64,6 +69,7 @@ export async function fetchArtists(brand) {
 * @returns {Promise<int|null>} - The fetched count of artists.
 */
 export async function fetchSongArtistCount(brand) {
+  const filter = new FilterBuilder()
```

Comment:
I don't know if this line of code is useful here.

Reply:
I wondered about that too 😂 but noticed this in Adrian's table in the PR comment:

<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">

<html>
<head>	
</head>

<body>

Method | Uses New Filter | User Raw Query | Hardcoded | Uses FetchAll | Has Test |  
-- | -- | -- | -- | -- | -- | --
fetchSongArtistCount | 1 |   | 1 |   | 1 |  


</body>

</html>
So I think it still needs to be updated to actually use the filter and to remove the hardcoded line below it

Reply:
You are right. But it was strange to see that line of code there, especially since I was only focusing on the 'Uses New Filter - true' column in the table :)

Reply:
I had originally planned to migrate it, but I think this is used in generic pages, not in catalogues, so it doesn't make sense to filter the extra information. 
It's been removed.